README.md

```diff
@@ -69,12 +68,39 @@ Please make sure to follow our code style and conventions.
 
 If you have any questions or suggestions about contributing, please feel free to contact us.
 
+## Local Development
+
+Follow these steps to set up a local development environment for Overlay using Google Chrome:
```

Comment:
I suggest the following steps:

(1. Clone and `cd`, I don't think it is needed, but if so, put them together).
2. Install dependencies by `yarn`.
3. Use `yarn build:watch` to rebuild the extension every time the files changed.
4. And open another terminal and run `yarn start:chrome` or `yarn start:firefox` to open the browser with the extension installed. Every time the extension files will change (by `build:watch`), the browser will reload the extension,